linux/Platform.c

```diff
@@ -107,6 +108,22 @@ static time_t Platform_Battery_cacheTime;
 static double Platform_Battery_cacheLevel = NAN;
 static ACPresence Platform_Battery_cacheIsOnAC;
 
+void Platform_init(void) {
+   if (access(PROCDIR, R_OK) != 0) {
+      fprintf(stderr, "Error: could not read procfs (compiled to look in %s).\n", PROCDIR);
+      exit(1);
+   }
+#ifdef HAVE_LIBSENSORS
```

Comment:
Add a blank line for better visual structure.

Reply:
+1